.github/workflows/notes.md

```diff
@@ -17,6 +35,19 @@ ${NVIM_VERSION}
 
 ### Linux (x64)
 
+#### Tarball
+
+1. Download **nvim-linux64.tar.gz**
+2. Extract: `tar xzvf nvim-linux64.tar.gz`
+3. Run `./nvim-linux64/bin/nvim`
+
+#### Debian Package
+
+1. Download **nvim-linux64.deb**
+2. Install the package using `sudo dpkg -i nvim-linux64.deb`
```

Comment:
```suggestion
2. Install the package using `sudo apt install ./nvim-linux64.deb`
```

Reply:
Can do, I'll make this change with any others in the same commit to address PR notes.